caps/management/commands/common.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
```

Comment:
presumably this can get dropped if #427 is merged first?

Reply:
also, given we're likely to be using this in a) data things, b) other climate projects I wonder if we should make a proper package with these functions 
we can add as a requirement?

Reply:
Yes, that's a good idea. I'll do that and update both PRs.

Reply:
@struan : both PRs now use the external `mysoc-dataset` package. Because of how I set the minimum requirements, I've had to bump the pandas from 1.3.4 to 1.3.5.  If you give me a thumbs up I can do a big merge of all these PRs.